scipy/stats/tests/test_continuous.py

```diff
@@ -915,6 +917,84 @@ def test_shapes(self):
         assert Y.mu[0] == 2
 
 
+class TestMakeDistribution:
+    @pytest.mark.parametrize('i, distdata', enumerate(distcont))
+    def test_make_distribution(self, i, distdata):
+        distname = distdata[0]
+
+        slow = {'argus', 'exponpow', 'exponweib', 'genexpon', 'gompertz', 'halfgennorm',
+                'johnsonsb', 'kstwobign', 'powerlognorm', 'powernorm', 'recipinvgauss',
+                'vonmises_line'}
+        if not int(os.environ.get('SCIPY_XSLOW', '0')) and distname in slow:
+            pytest.skip('Skipping as XSLOW')
+
+
+        if distname in {  # skip these distributions
+            'genpareto', 'genextreme', 'genhalflogistic',  # complicated support
+            'kstwo', 'kappa4', 'tukeylambda',  # complicated support
+            'levy_stable',  # levy_stable does things differently...
+            'ksone',  # tolerance issues
+            'norminvgauss',  # private methods seem to have broadcasting issues
+            'vonmises',  # circular distribution; shouldn't work
+            'irwinhall',  # requires dtype of shape parameter to be integer
+            'studentized_range',  # too slow
+        }:
```

Comment:
Do we know enough about the situations where `make_distribution` will fail to document them? 

I'm pretty sure parameter dependent support could work in principle. You can check if the `_get_support` method in the input distribution was overridden and work from there.

Is `levy_stable` problematic because it has class variables to control the parametrization and the pdf and cdf computation methods?

For `norminvgauss`, is the problem due to a bug in the original distribution?

Not supporting circular distributions is easy to document.

For `irwinhall`, is the issue specifically because it won't work with this particular test, or is requiring an integer for a shape parameter genuinely problematic?

Reply:
I don't remember.

Reply:
It might be coincidence, but I noticed that `genpareto`, `genextreme`, `kappa4` and `tukeylambda` with "complicated support" are all quantile-based distribution families, in the sense that it's usually mathematically easier to work with their QFs (or the derivatives, the QDFs) than with their CDFs. I believe that `kappa4` doesn't even have a closed-form CDF. I'm not sure how this could help with this, but I thought I'd let you know either way, as it might help later on somewhere 🤷🏻

Reply:
Those all have overrides for the `_cdf` and `_pdf`, so they should work once I override the generic method for determining the support. That is what holds them up right now ("complicated support").

(I haven't added a generic implementation of those functions in terms of the quantiles, though. So if they did not override `_pdf`, at least, that would be a problem.)

Reply:
> All of those things can be improved in time. For now, I planned to document that it doesn't work with these distributions.

Yeah, I think some kind of documentation that this won't always work is the only thing needed before we can merge this. How about?

`make_distribution` is experimental and may not work with all instances of `rv_continuous`. Known failures include <list of failures>.